service/controller/drainer.go

```diff
@@ -48,7 +47,6 @@ func NewDrainer(config DrainerConfig) (*Drainer, error) {
 			Logger:    config.Logger,
 			Resources: resources,
 			Selector: labels.SelectorFromSet(map[string]string{
-				key.PodWatcherLabel:   project.Name(),
```

Comment:
I don't see a use for this. Let me know if I'm missing anything.

Reply:
I think the risk here is that you might select pods which coincidentally have the operator version label.

Reply:
That makes a lot of sense. Unlikely, but doesn't hurt to keep it.